js/editor.ts

```diff
@@ -12,11 +12,11 @@ export enum ChangeType {
 
 interface EditorEvents {
   change: (contents: string, changeType: ChangeType) => void;
-  cursorMoved: (position: ace.Position, contents: string) => void;
+  cursorMoved: (position: ace.Ace.Point, contents: string) => void;
```

Comment:
If possible, please change the import so that this can be `ace.Point` instead of an ugly `ace.Ace.Point`.

Reply:
Not possible, that's how the ace definitions are laid out AFAIK